ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml

```diff
@@ -0,0 +1,891 @@
+base_images:
+  ansible:
+    name: "4.18"
+    namespace: ocp
+    tag: ansible
+  assisted-service-scripts:
+    name: ocm-2.11
+    namespace: edge-infrastructure
+    tag: assisted-service-scripts
+  aws-efs-csi-operator-create-efs:
+    name: "4.18"
+    namespace: ocp
+    tag: aws-efs-csi-operator-create-efs
+  cli:
+    name: "4.18"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.18"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+  tools:
+    name: "4.18"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.18.0-0
+        stream: 4-dev-preview
+        upper: 4.19.0-0
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-c2s-ipi-disc-priv-fips-f60
+  cron: 34 23 16 1,3,5,7,9,11 *
```

Comment:
For any job name changes or adding new jobs, we'd better run below command to update the cron settings:
`python ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py *your file path*`

Reply:
yes @JianLi-RH 
```
%   python3 ./tools/update-cron-entries.py openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml openshift-openshift-tests-private-release-4.18__arm64-nightly-4.18-upgrade-from-stable-4.18.yaml openshift-openshift-tests-private-release-4.18__multi-nightly-4.18-upgrade-from-stable-4.18.yaml 
Updating openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml
updating test job - aws-c2s-ipi-disc-priv-fips-f60
4 9 5 1,3,5,7,9,11 *
...
```

Reply:
sorry @JianLi-RH looks, cron updates were not committed, i will rerun and send new commit